src/main/java/uk/gov/companieshouse/documentsigningrequestconsumer/Consumer.java

```diff
@@ -41,7 +40,6 @@ public Consumer(Service service, MessageFlags messageFlags) {
             backoff = @Backoff(delayExpression = "${consumer.backoff_delay}"),
             dltTopicSuffix = "-error",
             dltStrategy = DltStrategy.FAIL_ON_ERROR,
-            fixedDelayTopicStrategy = FixedDelayStrategy.SINGLE_TOPIC,
```

Comment:
This should probably be replaced with https://docs.spring.io/spring-kafka/docs/3.0.10/api/org/springframework/kafka/retrytopic/SameIntervalTopicReuseStrategy.html rather than entirely removed, see https://docs.spring.io/spring-kafka/docs/3.0.9/api/org/springframework/kafka/retrytopic/FixedDelayStrategy.html

Reply:
RetryableTopic in org.springframework.kafka.annotation sets this value as default hence this as seen below
 SameIntervalTopicReuseStrategy sameIntervalTopicReuseStrategy() default SameIntervalTopicReuseStrategy.SINGLE_TOPIC;